demo-vue/src/mappers/vueStricklandMappers.js

```diff
@@ -0,0 +1,19 @@
+import { getValidationMessage, getValidationClassName } from '../../../demo/src/helpers';
+
+export default function mapFormFieldValidationStates (form) {
```

Comment:
This is a very subtle distinction, but I'd like to rename this to `vueValidationMappers`, removing "Strickland" from the name.  This is a concept that the application is introducing, not a concept that Strickland is introducing.  I think the name change could connote that.

Reply:
Yep, I see what you mean and it's a good distinction to make. I went this way because I see this being part a vue-strickland plugin eventually. Then I see the demo utilising that plugin. Would you mind the same names if they move over into a plugin? `vueStricklandXYZ`?